src/schema/v2/__tests__/startIdentityVerificationMutation.test.ts

```diff
@@ -0,0 +1,90 @@
+import { runQuery, runAuthenticatedQuery } from "schema/v2/test/utils"
+import { StartIdentityVerificationGravityOutput } from "lib/loaders/loaders_with_authentication/gravity"
+
+const mutation = `
+mutation {
+  startIdentityVerification(input: { identityVerificationId: "id-123"}) {
+    startIdentityVerificationResponseOrError {
+      ... on StartIdentityVerificationSuccess {
+        identityVerificationWizardUrl
+        identityVerificationId
+      }
+      ... on StartIdentityVerificationFailure {
+        mutationError {
+          type
+          message
+          detail
+        }
+      }
+    }
+  }
+}
+`
+
+describe("starting an identity verification", () => {
+  it("requires an access token", () => {
+    runQuery(mutation).catch(error => {
+      expect(error.message).toEqual(
+        "You need to be signed in to perform this action"
+      )
+    })
+  })
+
+  it("STUB: returns the given identity verification ID and a link to the staging auctions page", async () => {
```

Comment:
What is `STUB`?

Reply:
I believe this is just intended to communicate in the test docstring that the spec is not real or fully implemented yet..

Reply:
correct - just wanted to have a flag value in the specs that this isn't legit yet